packages/augur-sdk/src/api/GSN.ts

```diff
@@ -64,13 +64,20 @@ export class GSN {
       const signerAddress = await this.augur.dependencies.signer.getAddress();
       const walletAddress = await this.calculateWalletAddress(signerAddress);
       const wallet = this.augur.contracts.augurWalletFromAddress(walletAddress);
+
+      const useWallet = this.augur.getUseWallet();
```

Comment:
I didn't test this change as I'm not actually sure how to withdraw funds. @bthaile does this seem safe?